tests/01_06_02-pre-fingerroot500-release_request_received_bytes.js

```diff
@@ -1,62 +0,0 @@
-const expect = require('chai').expect
-const CustomError = require('../lib/CustomError');
-const rskUtils = require('../lib/rsk-utils');
-const { getBtcClient } = require('../lib/btc-client-provider');
-const { getRskTransactionHelpers } = require('../lib/rsk-tx-helper-provider');
-const { getDerivedRSKAddressInformation } = require('@rsksmart/btc-rsk-derivation');
-const btcEthUnitConverter = require('@rsksmart/btc-eth-unit-converter');
-const { sendTxToBridge } = require('../lib/2wp-utils');
-const { ensure0x } = require('../lib/utils');
-
-let btcTxHelper;
-let rskTxHelpers;
-
-describe('pegout events improvements - pre fingerroot', () => {
-
-  before(async () => {
-    btcTxHelper = getBtcClient();
-    rskTxHelpers = getRskTransactionHelpers();
-  });
-  
-  it('release_request_received event generates address as hash160', async () => {
```

Comment:
This test is not needed anymore after starting all the test with the latest fork.